Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -105,7 +105,7 @@ public extension String {
 		return String(first)
 	}
 
-	#if canImport(Foundation)
+    #if canImport(Foundation)
```

Comment:
Do we really need this check here? 🤔 I mean ... String is a type from the stdlib and to me since there's no bridging to NSString or usage of any foundation method it seems like there's no need  for that :))

Reply:
`String` actually has some extensions in `Foundation`, e.g. when using `Data`, which is part of `Foundation`, not the stdlib. Still, we should only be using this check for those situations where we use `Foundation` functions/classes.

Reply:
Hmmmm. Guess that doesn't hurt